ecosystem/typescript/sdk_v2/src/internal/account.ts

```diff
@@ -0,0 +1,106 @@
+import { AptosConfig } from "../api/aptos_config";
+import { AnyNumber } from "../bcs";
+import { get } from "../client";
+import { PaginationArgs } from "../client/types";
+import { Gen } from "../types";
+import { MaybeHexString, HexString, paginateWithCursor } from "../utils";
+
+export async function getData(aptosConfig: AptosConfig, accountAddress: MaybeHexString): Promise<Gen.AccountData> {
+  const { data } = await get<{}, Gen.AccountData>({
+    url: aptosConfig.network,
+    endpoint: `accounts/${HexString.ensure(accountAddress).hex()}`,
+    originMethod: "getData",
+    overrides: { ...aptosConfig.clientConfig },
+  });
+  return data;
+}
+
+export async function getModules(
+  aptosConfig: AptosConfig,
+  accountAddress: MaybeHexString,
+  query?: { ledgerVersion?: AnyNumber },
+): Promise<Gen.MoveModuleBytecode[]> {
+  const response = await paginateWithCursor<{}, Gen.MoveModuleBytecode[]>({
+    url: aptosConfig.network,
+    endpoint: `accounts/${HexString.ensure(accountAddress).hex()}/modules`,
+    params: { ledger_version: query?.ledgerVersion, limit: 1000 },
```

Comment:
Similarly why is limit set to 1000?  These should be provided by the user with a default that makes sense right?

We may want to expose the pagination the whole way up now that we can rewrite it